### ISSUE TITLE
Strange characters in the path pattern and comments.

### ISSUE THREAD

**Issue by onozaty** [onozaty/redmine-view-customize#76]
There are strange characters in the path pattern and comments.

![image](https://user-images.githubusercontent.com/47337/107841372-ba056000-6dfd-11eb-8307-3535a9c6cfdf.png)

* Redmine 3.4.x
* View customize 2.8.0

**Comment by onozaty** [comment]
Doesn't work with Rails 4?

* `007_change_path_pattern_default_on_view_customizes.rb` 
```ruby
class ChangePathPatternDefaultOnViewCustomizes < ActiveRecord::CompatibleLegacyMigration.migration_class
  def change
    change_column_default :view_customizes, :path_pattern, from: nil, to: ""
    change_column_default :view_customizes, :comments, from: nil, to: ""
    change_column_null :view_customizes, :comments, false, ""
  end
end
```

**Comment by onozaty** [comment]
Fixed at v2.8.1
